libs/core/src/components/sage-tab/sage-tab.tsx

```diff
@@ -0,0 +1,76 @@
+import { Component, Element, Host, h, Prop, Event, EventEmitter } from '@stencil/core';
+
+@Component({
+  tag: 'sage-tab',
+  styleUrl: 'sage-tab.scss',
+  shadow: false,
+})
+export class SageTab {
+  @Element() el: HTMLElement;
+
+  /**
+   * Sets the related tab name, this name must match a `sage-tabpanel`'s tab property
+  */
+  @Prop() tab!: string;
+
+  /**
+   * Keeps track of the parentComponentId unique id, this property is passed by parent component
+  */
+  /** @internal */
+  @Prop({mutable: true}) parentComponentId: string;
+
+  /**
+   * Keeps track of if the expected tab variant, this property is passed by parent component
+  */
+  /** @internal */
+  @Prop() variant: string;
+
+  /**
+   * Keeps track of if the expected tab variant, this property is passed by parent component
+  */
+  /** @internal */
```

Comment:
Looks like you need to update the description of the prop here. Probably from a copy and paste.